### ISSUE TITLE
Move to 7.6.9 appears to not have created a publicly available image on docker-hub

### ISSUE THREAD

**Issue by michael-beaver** [stefancrain/folding-at-home#9]
Thanks for making this build! Super nice to be able to containerize this on my homelab.

I see that you moved the [version file](https://github.com/stefancrain/folding-at-home/blob/master/build/.version) over to 7.6.9, but there doesn't appear to be a new tag on [dockerhub](https://hub.docker.com/r/stefancrain/folding-at-home/tags). The version there it still is at 7.5.1 and hasn't been updated recently. Perhaps something went wrong with the Github workflow?

**Comment by stefancrain** [comment]
@michael-beaver Well spotted, thanks for opening an issue!
It looks like 7.6.9 was released 9 days ago. I'll have a manual release shortly. 

The workflow requires an update to https://formulae.brew.sh/api/cask/folding-at-home.json (_which was the cleanest, mosts consumable version number I could find_) which currently lists `7.6.9` ... so it should have updated.

Will post more detailed analysis ASAP.

**Comment by stefancrain** [comment]
Thanks again for reporting @michael-beaver. 

**7.6.9** was correctly updated by the workflow schedule but a build was not triggered. This happened because of the way I was pushing to the repo from within the workflow. Today I learned ([from @xUnholy](https://github.com/raspbernetes/multi-arch-images/blob/07739013c5ba8deba0ccc75d9c00863b55fbfa28/.github/workflows/schedule.yml#L97), THANKS!) that you can not use `git` within Github workflows to push to itself and trigger _downstream_ builds. 

In short, Github is protecting its workflow service from infinite recursion, builds causing builds * ∞

I had to break the `checkout` and `push` commands into their own action with `checkout` and `github-push-action`

**Comment by michael-beaver** [comment]
Wow, thanks for getting to this so quickly! Unfortunately it appears the new image is breaking the ability for my setup to see GPUs anymore, nothing in my configuration changed other than the image tag, and reverting to 7.5.1 allows the GPUs to be utilized again. is this happening to you as well or is it just me?

Very weird as I can see in the git history that you clearly didn't change much with the build. Maybe the simultaneous uplift of the tini version caused issues?

**Comment by stefancrain** [comment]
@michael-beaver oh no that's not good! I only have my laptop with me currently with no access to GPU nodes to test. It's possible tini could have burnt us, let me revert that. One thing at a time :-). 

- [The release notes](https://foldingforum.org/viewtopic.php?f=24&t=34466) mention some GPU.txt changes. 
- I see others [reporting similar issues](https://www.reddit.com/r/Folding/comments/g3ghyi/new_client_released_769/) 
  - > New client is not recognising my GPU

What does these commands show you? 
```
docker run \
 --rm \
 --gpus all \
 --entrypoint="nvidia-smi" \
 stefancrain/folding-at-home:latest
```
And
```
docker run \
 --rm \
 --gpus all \
 --entrypoint="/usr/bin/FAHClient" \
 stefancrain/folding-at-home:latest \
 --info

```

**Comment by stefancrain** [comment]
I've verified the version upgrade schedule works as expected, and re-released `7.6.9` with tini 0.18.0

**Comment by michael-beaver** [comment]
Thanks for the support on this. The newest image did not appear to resolve anything, so the tini version was probably a red herring.

The new version worked great on my windows 10 machine so I missed that other people were running into somewhat similar problems, that could absolutely point to it being an upstream issue with F@H. Will need to do some more debugging with this to figure out what is wrong. Currently struggling to get the FAHControl client to connect to my docker instance to try some of things noted in that thread.


Here is the output you asked for, all looks pretty normal as far as I can tell. nvidia-smi is working the same as it does on the host and as it did in 7.5.1.
```
$ docker run \
>  --rm \
>  --gpus all \
>  --entrypoint="nvidia-smi" \
>  stefancrain/folding-at-home:latest
Tue Apr 28 05:43:21 2020
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.33.01    Driver Version: 440.33.01    CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 970     On   | 00000000:01:00.0 Off |                  N/A |
| 31%   58C    P0    52W / 163W |     13MiB /  4042MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce GTX 970     On   | 00000000:04:00.0 Off |                  N/A |
|  0%   39C    P8    14W / 163W |     13MiB /  4043MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
```

```
$ docker run \
>  --rm \
>  --gpus all \
>  --entrypoint="/usr/bin/FAHClient" \
>  stefancrain/folding-at-home:latest \
>  --info
********************************** FAHClient ***********************************
        Version: 7.6.9
         Author: Joseph Coffland <joseph@cauldrondevelopment.com>
      Copyright: 2020 foldingathome.org
       Homepage: https://foldingathome.org/
           Date: Apr 17 2020
           Time: 18:11:26
       Revision: 398c2b17fa535e0cc6c9d10856b2154c32771646
         Branch: master
       Compiler: GNU 8.3.0
        Options: -std=c++11 -ffunction-sections -fdata-sections -O3
                 -funroll-loops -fno-pie
       Platform: linux2 4.19.0-5-amd64
           Bits: 64
           Mode: Release
           Args: --info
************************************ CBang *************************************
           Date: Apr 17 2020
           Time: 18:10:13
       Revision: 2fb0be7809c5e45287a122ca5fbc15b5ae859a3b
         Branch: master
       Compiler: GNU 8.3.0
        Options: -std=c++11 -ffunction-sections -fdata-sections -O3
                 -funroll-loops -fno-pie -fPIC
       Platform: linux2 4.19.0-5-amd64
           Bits: 64
           Mode: Release
************************************ System ************************************
            CPU: AMD FX(tm)-8350 Eight-Core Processor
         CPU ID: AuthenticAMD Family 21 Model 2 Stepping 0
           CPUs: 8
         Memory: 31.44GiB
    Free Memory: 341.68MiB
        Threads: POSIX_THREADS
     OS Version: 4.15
    Has Battery: false
     On Battery: false
     UTC Offset: 0
            PID: 1
            CWD: /app
             OS: Linux 4.15.0-91-generic x86_64
        OS Arch: AMD64
           GPUs: 0
  CUDA Device 0: Platform:0 Device:0 Bus:1 Slot:0 Compute:5.2 Driver:10.2
  CUDA Device 1: Platform:0 Device:1 Bus:4 Slot:0 Compute:5.2 Driver:10.2
OpenCL Device 0: Platform:0 Device:0 Bus:1 Slot:0 Compute:1.2 Driver:440.33
OpenCL Device 1: Platform:0 Device:1 Bus:4 Slot:0 Compute:1.2 Driver:440.33
************************************ libFAH ************************************
           Date: Apr 15 2020
           Time: 21:43:24
       Revision: 216968bc7025029c841ed6e36e81a03a316890d3
         Branch: master
       Compiler: GNU 8.3.0
        Options: -std=c++11 -ffunction-sections -fdata-sections -O3
                 -funroll-loops -fno-pie
       Platform: linux2 4.19.0-5-amd64
           Bits: 64
           Mode: Release
********************************************************************************
```

**Comment by stefancrain** [comment]
Thanks for the info, I'm a little at a loss as how to proceed as F@H reports it can see your CPUS, GeForce GTX 970 is [in the supported cpu list](https://apps.foldingathome.org/GPUs.txt)

>CUDA Device 0: Platform:0 Device:0 Bus:1 Slot:0 Compute:5.2 Driver:10.2
>CUDA Device 1: Platform:0 Device:1 Bus:4 Slot:0 Compute:5.2 Driver:10.2
>OpenCL Device 0: Platform:0 Device:0 Bus:1 Slot:0 Compute:1.2 Driver:440.33
>OpenCL Device 1: Platform:0 Device:1 Bus:4 Slot:0 Compute:1.2 Driver:440.33
 
>Unfortunately it appears the new image is breaking the ability for my setup to see GPUs anymore

Maybe you can share the error messages you are seeing?

**Comment by stefancrain** [comment]
@michael-beaver 7.6.13 was released (automatically!) and I think it's fair to close this issue as the githab workflow is running smoothly. 

Please give this a try and see if it solves your issues. 

```bash
docker pull stefancrain/folding-at-home:7.6.13
```
